hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/HoodieIncrSource.java

```diff
@@ -128,6 +133,10 @@ public static class Config {
   public HoodieIncrSource(TypedProperties props, JavaSparkContext sparkContext, SparkSession sparkSession,
                           SchemaProvider schemaProvider) {
     super(props, sparkContext, sparkSession, schemaProvider);
+
+    this.snapshotLoadQuerySplitter = Option.ofNullable(props.getString(SNAPSHOT_LOAD_QUERY_SPLITTER_CLASS_NAME, null))
+        .map(className -> (SnapshotLoadQuerySplitter) ReflectionUtils.loadClass(className,
+              new Class<?>[]{TypedProperties.class}, props));
```

Comment:
what happens if className is null or class does not exist?

Reply:
className is null then this.snapshotLoadQuerySplitter = Option.empty() . 
class does not exist -> will throw class not found error .

Reply:
yeah this will fallback to regular pipeline for null , does not exist we should keep i think , ( could be jar issue ) .